crates/backend/src/ast.rs

```diff
@@ -201,21 +203,39 @@ impl Program {
             panic!("can only bindgen safe functions");
         }
 
-        let opts = BindgenAttrs::find(&mut method.attrs);
+        let mut opts = BindgenAttrs::find(&mut method.attrs);
+        let is_constructor = opts.constructor();
+        let constructor = if is_constructor {
+            Some(method.sig.ident.to_string())
+        } else {
+            None
+        };
+
+        if is_constructor {
+            let pos = opts.attrs
+                .iter()
+                .enumerate()
+                .find(|(_, a)| **a == BindgenAttr::Constructor)
+                .unwrap()
+                .0;
+            opts.attrs.remove(pos);
```

Comment:
How come this was necessary to remove the attribute from our list?

Reply:
This was a failed attempt to remove the `#[wasm_bindgen(constructor)]` from the function; It's removed now.